include/dlaf/matrix/matrix_view_const.tpp

```diff
@@ -8,6 +8,8 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+// TODO: Remove completely (and rewrite later)?
+
```

Comment:
@rasolca what would be your preference regarding `MatrixView`: would you prefer to keep it or could be removed for now and recreated when it's needed?

Reply:
Fine with removal if it doesn't break anything.

Reply:
It's been removed for now.